exercises/binary-search-tree/src/test/java/BinarySearchTreeTest.java

```diff
@@ -6,15 +6,21 @@
 import static org.junit.Assert.assertNotNull;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.Before;
 
-public class BSTTest {
+public class BinarySearchTreeTest {
+    BinarySearchTree<Integer> bst;
+
+    @Before
+    public void setUp() {
+        bst = new BinarySearchTree<>();
```

Comment:
I would probably rename this as well to `binarySearchTree` because I find that clearer. That's just my preference though, feel free to leave it as it is if you prefer it that way :)